curp/src/rpc/mod.rs

```diff
@@ -489,3 +482,17 @@ impl ConfChange {
         }
     }
 }
+
+impl ShutdownResponse {
+    /// Create a new shutdown response
+    pub(crate) fn new(leader_id: Option<ServerId>, term: u64, error: Option<ProposeError>) -> Self {
+        let error = error.map(|e| PbProposeErrorOuter {
+            propose_error: Some(e.into()),
+        });
```

Comment:
Could we implement `From` trait for PbProposeErrorOuter type?

Reply:
Fixed